pkg/gitversion/convert_test.go

```diff
@@ -0,0 +1,116 @@
+package gitversion
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+type versionTest struct {
+	desc   string
+	semver string
+	python string
+}
+
+func TestConversions(t *testing.T) {
+	inputs := []versionTest{
+		{
+			desc:   "Repo with exact tag",
+			semver: "1.0.0",
+			python: "1.0.0",
+		},
+		{
+			semver: "0.0.0",
+			python: "0.0.0",
+		},
+		{
+			desc:   "Repo with no tags",
+			semver: "0.0.1-alpha.0+68804cfa",
+			python: "0.0.1a0",
+		},
+		{
+			desc:   "Repo with no tags",
+			semver: "0.0.1-alpha.0+68804cfa",
+			python: "0.0.1a0",
+		},
+		{
+			desc:   "Repo with with commit after tag",
+			semver: "0.0.1-alpha.0+68804cfa",
+			python: "0.0.1a0",
+		},
```

Comment:
This looks the same as the previous case's test data?

Reply:
Yup - copy paste errors taking it from the gitversion tests 👍